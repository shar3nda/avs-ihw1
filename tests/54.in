9
9 10 -1 9 7 1 10 2 -2