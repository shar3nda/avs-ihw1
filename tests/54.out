-7 -9 -10 -6 -8 -5 6 9 6